src/connections.jl

```diff
@@ -318,6 +318,34 @@ end
 
 feedback2dof(B,A,R,S,T) = tf(conv(B,T),zpconv(A,R,B,S))
 
+"""
+    feedback2dof(P::TransferFunction, C::TransferFunction, F::TransferFunction)
+
+Return the transfer function
+`P(F+C)/(1+PC)`
+which is the closed-loop system with process `P`, controller `C` and feedforward filter `F` from reference to control signal (by-passing `C`).
+
+         +-------+
+         |       |
+   +----->   F   +----+
+   |     |       |    |
+   |     +-------+    |
+   |     +-------+    |    +-------+
+r  |  -  |       |    |    |       |    y
++--+----->   C   +----+---->   P   +---+-->
+      |  |       |         |       |   |
+      |  +-------+         +-------+   |
+      |                                |
+      +--------------------------------+
+"""
+function feedback2dof(P::TransferFunction{T}, C::TransferFunction{T}, F::TransferFunction{T}) where T
+    !issiso(P) && error("Feedback not implemented for MIMO systems")
+    Pn,Pd = numpoly(P)[], denpoly(P)[]
+    Cn,Cd = numpoly(C)[], denpoly(C)[]
+    Fn,Fd = numpoly(F)[], denpoly(F)[]
+    den = (Cd*Pd + Pn*Cn)*Fd
+    isdiscrete(P) ? tf(Cd*Pn*Fn + Pn*Cn*Fd, den, P.Ts) : tf(Cd*Pn*Fn + Pn*Cn*Fd, den)
+end
```

Comment:
I hope the following works, I think it's how @mfalt and I tried to use it for other types of connections. It handles promotion of sample times and checks for sample-time mismatch.
```suggestion
function feedback2dof(P::TransferFunction{TE}, C::TransferFunction{TE}, F::TransferFunction{TE}) where TE
    !issiso(P) && error("Feedback not implemented for MIMO systems")
    timeevol = common_timeevol(P, C, F)
    
    Pn,Pd = numpoly(P)[], denpoly(P)[]
    Cn,Cd = numpoly(C)[], denpoly(C)[]
    Fn,Fd = numpoly(F)[], denpoly(F)[]
    den = (Cd*Pd + Pn*Cn)*Fd
    tf(Cd*Pn*Fn + Pn*Cn*Fd, den, timeevol)
end
```